cookbook/email/gmail.rst

```diff
@@ -55,33 +50,84 @@ In the development configuration file, change the ``transport`` setting to
             'password'  => 'your_gmail_password',
         ));
 
-You're done!
+If you are using the Symfony Standard Edition, it's more convenient to configure
```

Comment:
it's the cookbook, so we can assume they are and we can say "It's more convenient to configure these options in the `parameters.yml.dist` file:"